### ISSUE TITLE
not accessible with http

### ISSUE THREAD

**Issue by xzonz** [telmomarques/xiaomi-360-1080p-hacks#58]
Hello , thanks for the hack , i was waiting for it since months !!

i've downgraded the firmware -> ok  (verified on the android app mi home)

i've loaded the hack following the instructions , but , impossible to connect with http
browser said : '' ERR_CONNECTION_REFUSED ''
i've tried ssh with putty : same result.

i've tried with two differents sdcard !!
any suggestions doctor ? :-)

Thanks for your help

**Comment by telmomarques** [comment]
Hi,

Please confirm you downloaded the latest release and not the source code.

**Comment by xzonz** [comment]
hi , i took the **v1.2-rc1** ... right ?

**Comment by telmomarques** [comment]
Yes, but make sure you downloaded "sdcard.zip" and not the source code.

**Comment by xzonz** [comment]
ok , sure that's what i've downloaded.
i put the whole folders (framegrabber , rstp-server ..... )
at the root of the sdcard and followed the instruction.

**Comment by telmomarques** [comment]
The folders "hacks" and "manu_test" should go no SDCard root.

**Comment by xzonz** [comment]
i've just put those ones at the root

![image](https://user-images.githubusercontent.com/23436761/91886961-63123100-ec8a-11ea-84f5-4e42bb2c1e96.png)

**Comment by telmomarques** [comment]
That's the problem 🙂
You should unzip sdcard.zip directly on the SDCard.

**Comment by xzonz** [comment]
ok i will make the test tonight

thanks very much for your help !  👍

**Comment by pedromalima** [comment]
Hi,

I have a similar issue, I can view the feed from MI app, but when scanning through NMAP it does not work. all ports closed.
I got the following debug info
[miio_client.log](https://github.com/telmomarques/xiaomi-360-1080p-hacks/files/5250469/miio_client.log)
[diagnosis.txt](https://github.com/telmomarques/xiaomi-360-1080p-hacks/files/5250470/diagnosis.txt)
[messages.txt](https://github.com/telmomarques/xiaomi-360-1080p-hacks/files/5250472/messages.txt)

Can you please have a look?
the downgrade worked fine, thanks

**Comment by titusfx** [comment]
> ok i will make the test tonight
> 
> thanks very much for your help !

Any luck?

**Comment by Spirineu** [comment]
Hi, 

Thank you for this hack.

But I'm really new to HA, can you help me out?

Ive put the files on the sd card but I cant acess the web config using my I.P 😞 

![image](https://user-images.githubusercontent.com/72891552/96043473-ec5e7b00-0e66-11eb-817b-e9e2b75b29fc.png)

**Comment by Rtamagnini** [comment]
my camera works at ha with rtsp.
how to wotk via http ? 

http://192.168.1.104:8554/mainstream ?

**Comment by Vorlex** [comment]
@Rtamagnini 
![image](https://user-images.githubusercontent.com/24228944/99516916-68cd0800-2997-11eb-887e-9e62e953602f.png)
